stepflow-rs/crates/stepflow-builtins/src/blob.rs

```diff
@@ -71,13 +71,18 @@ impl BuiltinComponent for PutBlobComponent {
 
         let data_ref = ValueRef::new(input.data);
 
+        // DEBUG: Log what's being stored
+        tracing::debug!("put_blob storing data: {:?}", data_ref.as_ref());
+
         // Create the blob through the execution context
         let blob_id = context
             .state_store()
             .put_blob(data_ref)
             .await
             .change_context(BuiltinError::Internal)?;
 
+        tracing::debug!("put_blob created blob with ID: {}", blob_id.as_str());
```

Comment:
Ditto -- I think the `blob_id` should wokr here (may need `{blob_id:?}` if we haven't implemented display, although we should support that if not).